src/create-form-view.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+export default class CreateFormView {
+  constructor(element, application) {
+    this.element = element;
+    this.application = application;
+    this.togglePuppyForm();
+    this.puppyAddToList();
+  }
+
+  puppyAddToList() {
+    this.element.querySelector(`.dropdown-button`).addEventListener(`click`, (ev) => {
```

Comment:
This action should trigger when the form submits not just when the button is pressed.